packages/@ember/-internals/glimmer/lib/environment.ts

```diff
@@ -100,6 +106,15 @@ const VM_DEPRECATION_OVERRIDES: (DeprecationOptions & { message?: string })[] =
       enabled: '3.21.0',
     },
   },
+  {
+    id: 'this-property-fallback',
+    disabled: ENV._DISABLE_PROPERTY_FALLBACK_DEPRECATION,
+    until: '4.0.0',
+    for: 'ember-source',
```

Comment:
The `url` for this deprecation will be https://deprecations.emberjs.com/v3.x#toc_this-property-fallback, based on https://github.com/ember-learn/deprecation-app/pull/773

Reply:
Thanks! updated